### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
-  3.10.4 (#42)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -114,17 +119,15 @@ phpversion=$(ynh_app_setting_get --app=$app --key=phpversion)
 # SECURE FILES AND DIRECTORIES
 #=================================================
 
-ynh_app_setting_set --app=$app --key=data_path --value=$data_path
 mkdir -p $data_path
-chown -R $app: $data_path
-chown -R $app: $final_path
+chown -R $app:www-data "$data_path"
```
Comment (yalh76) [inline scripts/install:123]: Maybe you should manage rights on the `$data_path` directory

File: scripts/restore [hunk 2]
```diff
@@ -52,31 +52,28 @@ ynh_script_progression --message="Restoring the NGINX configuration..." --weight
...
-ynh_script_progression --message="Restoring user rights..."
+ynh_script_progression --message="Restoring Moodle main directory..." --weight=1
+
+ynh_restore_file --origin_path=$final_path
+
+chmod 750 "$final_path"
+chmod -R o-rwx "$final_path"
+chown -R $app:www-data "$final_path"
+
+ynh_restore_file --origin_path=$data_path
```
Comment (yalh76) [inline scripts/restore:74]: Would be cool to use `--is_big` during backup and `--not_mandatory` during restore.
Doing that `$data_path` won't be backup during ynh_backup_before_upgrade in the upgrade script; making upgrade quicker

**Review by yalh76 — APPROVED** [review]
Maybe few enhancements around `$data_path` to do ^^

**Comment by ericgaspar** [comment]
!testme

**Comment by kay0u** [comment]
in the CI logs:
> 61019 WARNING PHP Warning:  chdir(): Permission denied (errno 13) in /var/www/moodle/admin/cli/install.php on line 792

**Comment by ericgaspar** [comment]
!testme

**Review by yalh76 — APPROVED** [review]

File: doc/DISCLAIMER.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+## Configuration
+
+How to configure this app: by an admin panel.
+
+## YunoHost specific features
+
+#### Multi-user support
+
+* Are LDAP and HTTP auth supported? **Yes** 
+* Can the app be used by multiple users? **Yes**
```
Comment (alexAubin) [inline doc/DISCLAIMER.md:10]:
```suggestion
- This app supports LDAP / SSO integration with Yunohost
```

File: doc/DISCLAIMER_fr.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+## Configuration
+
+* Comment configurer cette application : via le panneau d'administration.
+
+## Caractéristiques spécifiques YunoHost
+
+#### Support multi-utilisateur
+
+* L'authentification LDAP et HTTP est-elle prise en charge ? **Oui**
+* L'application peut-elle être utilisée par plusieurs utilisateurs ? **Oui**
```
Comment (alexAubin) [inline doc/DISCLAIMER_fr.md:10]:
```suggestion
- Cette app est intégrée avec le LDAP / SSO de Yunohost
```

**Review by alexAubin — APPROVED** [review]

File: scripts/upgrade [hunk 5]
```diff
@@ -70,10 +78,29 @@ if [ "$upgrade_type" == "UPGRADE_APP" ]
...
+	ynh_secure_remove --file="$final_path"
+
 	# Download, check integrity, uncompress and patch the source from app.src
 	ynh_setup_source --dest_dir="$final_path"
+
+	# Copy the admin saved settings from tmp directory to final path  
+  	cp -a "$tmpdir/config.php" "$final_path/config.php" 
+
+  	# Remove the tmp directory securely
+  	ynh_secure_remove --file="$tmpdir"
```
Comment (alexAubin) [inline scripts/upgrade:97]: I'm not sure but you may be able to simplify this with (assuming Yunohost >= 4.2) :

```suggestion
	# Download, check integrity, uncompress and patch the source from app.src
	ynh_setup_source --dest_dir="$final_path" --keep="$final_path/config.php"
```

Comment (ericgaspar) [reply scripts/upgrade:97]: I have tried, but the app seems to complain about mixed version files...

**Pull request merged**